pkg/controller/amazoncloudintegration/amazoncloudintegration_controller.go

```diff
@@ -200,6 +208,11 @@ func (r *ReconcileAmazonCloudIntegration) Reconcile(request reconcile.Request) (
 	return reconcile.Result{}, nil
 }
 
+func (r *ReconcileAmazonCloudIntegration) SetDegraded(reason string, err error, log logr.Logger) {
+	log.Error(err, reason)
```

Comment:
Should this be a warning?
```suggestion
	log.Warn(err, reason)
```